src/System.Security.Cryptography.X509Certificates/tests/FindTests.cs

```diff
@@ -369,13 +369,33 @@ public static void TestBySubjectKeyIdentifier_MatchA()
                 "B4D738B2D4978AFF290A0B02987BABD114FEE9C7");
         }
 
-        [Fact]
+        [Theory]
+        [InlineData("5971A65A334DDA980780FF841EBE87F9723241F2")]
+        // Whitespace is allowed
+        [InlineData("59 71\tA6 5A 33 4D DA 98 07 80 FF 84 1E BE 87 F9 72 32 41 F2")]
```

Comment:
@JonHanna Seems pretty reasonable, but I think I'll bring that in for a successive change (unless CI keeps having issues, in which case I might as well just improve this one)